action.yml

```diff
@@ -1,33 +1,105 @@
 ---
-name: "Lazy Action Dotnet"
-description: "Github action to perform build, test , scan and generate image."
+# yamllint disable rule:line-length
+name: Lazy Action Dotnet
+description: |
+  Github Action to perform build, test , scan and generate Image for Dotnet
+
+  ## Permissions
+
+  Add the following permissions to the job
+
+  ```yaml
+  permissions:
+    id-token: write
+    contents: read
+  ```
+
+  ## Usage
+
+  ```yaml
+      - name: Actions Dotnet
+        uses: variant-inc/actions-dotnet@v2
+        with:
+          ecr_repository: 'demo/example'
+          dotnet-version: '6.0.x'
+  ```
 inputs:
-  src_file_dir_path:
-    description: "Directory path to the solution file"
-    required: true
-    default: "."
+  aws_region:
+    description: Region where the image will be created.
+    required: false
+    default: us-east-2
   dockerfile_dir_path:
-    description: "Directory path to the dockerfile"
-    required: true
+    description: Directory path to the dockerfile
+    required: false
+    default: .
+  dotnet-version:
```

Comment:
Rename to `dotnet_version` to have same casing for inputs.

Reply:
I get this but actions-python and nodejs have it this way because [actions/setup-dotnet](https://github.com/actions/setup-dotnet#usage) uses an input with this casing so this is to keep it uniform with that action